cmd/leanix-k8s-connector/main.go

```diff
@@ -99,16 +90,20 @@ func KubernetesScan(debugLogBuffer *bytes.Buffer) (response *leanix.SelfStartRes
 
 	log.Info("----------Start----------")
 	log.Infof("LeanIX Kubernetes connector build version: %s", version.VERSION)
-	log.Infof("LeanIX integration version: %s", lxVersion)
-	log.Infof("LeanIX connector id: %s", lxConnectorID)
-	log.Infof("LeanIX connector type: %s", lxConnectorType)
-	log.Infof("LeanIX connector processing direction: %s", lxConnectorProcessingDirection)
+	log.Infof("LeanIX integration version: %s", startResponse.BindingKey.LXVersion)
+	log.Infof("LeanIX connector id: %s", startResponse.BindingKey.ConnectorId)
+	log.Infof("LeanIX connector type: %s", startResponse.BindingKey.ConnectorType)
+	log.Infof("LeanIX connector processing direction: %s", startResponse.BindingKey.ProcessingDirection)
 	log.Infof("Target LeanIX workspace: %s", viper.GetString(lxWorkspaceFlag))
 	log.Infof("Target Kubernetes cluster name: %s", startResponse.ConnectorConfiguration.ClusterName)
 
 	var config *restclient.Config
 	if viper.GetBool(localFlag) {
-		config, err = clientcmd.BuildConfigFromFlags("", filepath.Join(homedir.HomeDir(), ".kube", "config"))
+		if viper.GetString(kubeConfigPathFlag) != "" {
```

Comment:
Sounds like kube-config-path is a new parameter. Did we already adapt the helm chart command to pass in this parameter or where does it come from when running in customers env?

Reply:
Ah good point, no that is still missing, honestly Im also still not sure if this parameter is really worth adding. I use it for local development so I can have a small clean kube config file in a separate location only for that. So I would also be open to keep this parameter out.

Reply:
We can still keep this in but as an addition to the previous `local` flag. Or we go ahead and figure out where a kubernetes config normally exists in a customers environment

Reply:
Well usually the kubeconfig is found in the .kube folder as "config", and this is already part of the local flag as of now. So the only value this really adds is being able to have a separate kube config just for testing the k8s connector.

Reply:
Sleeping over it, I will just remove it, as it probably doesnt serve much purpose for most people, especially customers.

Reply:
Sorry for being little late for the party. Why do we need kubeconfig file as such? Is it because if you want to have clustername from it. But it has certificates as well, it may not be a value add for customer.

Reply:
The kubeconfig is used for local testing, as you can use it to retrieve cluster information via the api from outside the cluster. I implemented this additional flag as I wanted to be able to save a separate kubeconfig outside of the default folder that its found in, but I will remove it as I think the usecase is fairly limited.